### ISSUE TITLE
Installing NewsCRUD as a Package (Option B)

### ISSUE THREAD

**Issue by jmichaelterenin** [Laravel-Backpack/NewsCRUD#39]
# Bug report

### What I did:
I've purchased a commercial licensed version of Laravel Backpack 4.0.23.
I have an existing project where I've been using cviebrock\eloquent-sluggable for some time now. I recently upgraded it to v4.8

### What I expected to happen:
I wanted to install the NewsCRUD package, and assumed it would install as Laravel Backpack did, without any problems.

### What happened:

```
Using version ^3.0 for backpack/newscrud
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for backpack/newscrud ^3.0 -> satisfiable by backpack/newscrud[3.0.0].
    - backpack/newscrud 3.0.0 requires cviebrock/eloquent-sluggable ^6.0 -> satisfiable by cviebrock/eloquent-sluggable[6.0.0, 6.0.1, 6.0.2] but these conflict with your requirements or minimum-stability.

```
### What I've already tried to fix it:

I've searched in Issues for anyone experiencing this problem, but nothing came up.

### Backpack, Laravel, PHP, DB version:
Backpack 4.0.23, Laravel 5.8, PHP 7.2.24

**Comment by pxpm** [comment]
Hello @jmichaelterenin 

Thanks for interest in Backpack. The problem is clearly stated: you are using an unsuported version of `cviebrock/eloquent-sluggable` 

Can you update `cviebrock/eloquent-sluggable` to version `6.0^` or completely delete it from composer so backpack news will require the correct version to work with. 

Best,
Pedro

**Comment by jmichaelterenin** [comment]
But version 6.0 of civebrock/eloquent-sluggable is for Laravel 6, and says so in its documentation.
I am using Laravel 5.8. I cannot currently upgrade to 6 because this is an existing project that is using packages not yet available for Laravel 6. 

I bring this up here because the NewCrud documentation states that it's available for Laravel 5.8.

**Comment by jmichaelterenin** [comment]
@tabacitu, can you please provide some insight? Is this NEWSCrud package really available for Laravel 5.8, and if so, what version should I be pulling so that there's no dependency issue with cviebrock/eloquent-sluggable?

**Comment by tabacitu** [comment]
@jmichaelterenin we're on it. 

@pxpm came up with a possible solution: for us to require version ```6.0|5.8``` in our ```composer.json```. He'll be working on draft fix ASAP, and get back to you to test it. If it works for you, we'll merge it for everyone.

Cheers!

**Comment by pxpm** [comment]
Hello @jmichaelterenin !

Just created a fresh laravel 5.8 project and added NewsCRUD. Could you please test if it's working on your side ? 

I will need you to do: 

`composer require backpack/newscrud:dev-dev-version`

Instead of the require that is presented in intallation instructions. 

Please let me know if it works for you so we can push this fix to everybody :) 

Best,
Pedro

**Comment by jmichaelterenin** [comment]
@pxpm, @tabacitu I apologize for the delay, I had some personal things to attend to.
I was able to complete installation (Installation type (B) - package) ,and although I haven't had the time just yet to add any dummy/test data to see it in action, I have to assume we got past the issue that presented itself originally. 

Thank you for your assistance, and taking the time to adjust things for Laravel 5.8. I want to be able to eventually move our existing Admin to Backpack, and once I've done so, we'll upgrade to 6.0. That will take a couple of months though, so again, I appreciate your efforts.

I will let either one of you close out this thread.
